post.md

```diff
@@ -0,0 +1,503 @@
+---
+post_title: Flashing eMMC on Hummingboard Edge using fastboot - part 1
+author: Maciej Pijanowski
+post_excerpt: ""
+layout: post
+published: true
+post_date: 2018-01-23 12:00:00
+tags:
+  - U-Boot
+  - eMMC
+  - fastboot
+  - Hummingboard
+  - i.MX6
+categories:
+  - manufacturing
+  - firmware
+---
+
+# Introduction
+
+Flashing an eMMC of produced board is one of the crucial manufacturing
+procedures. This post series presents how one can take advantage of i.MX6
+features and open source tools to prepare themselves with quite robust and easy
+to use process.
+
+Target reference platform is
+[Hummingboard Edge](https://www.solid-run.com/product/hummingboard-edge-imx6d-0c-e/https://www.solid-run.com/product/hummingboard-edge-imx6d-0c-e/).
+
+# General concept
+
+The general concept is inspired by
+[this great FreeElectrons post:](https://free-electrons.com/blog/factory-flashing-with-u-boot-and-fastboot-on-freescale-i-mx6/)
```

Comment:
@macpijan Free Electrons had to change name because of some legal problems to Bootlin